docker/Dockerfile

```diff
@@ -14,7 +14,6 @@ ARG cxx_abi="1"
 ARG tpuvm=""
 ARG bazel_jobs=""
 ARG git_clone="true"
-ARG build_cpp_tests="0"
 
 ENV BUILD_CPP_TESTS "${build_cpp_tests}"
```

Comment:
Seems like we deleted all the cpp tests related flag, is this `BUILD_CPP_TESTS` used anywhere now?